src/Finch_Run.hpp

```diff
@@ -110,13 +110,13 @@ class Layer
 
     auto writeSolidificationData() { return solidification_data_.write(); }
 
-    std::array<double, 3> getLowerSolidificationDataBounds()
+    std::array<double, 3> getLowerSolidificationDataBounds( MPI_Comm comm )
```

Comment:
Conflicts with ExaCA/bin/runCoupled.cpp line 53 (which uses a call to this function without input arguments)

Reply:
Forgot to push my local changes where I marked the previous as deprecated